### PR TITLE
More! borg buffing/qol and gifts for prospecting.

### PR DESCRIPTION
**Pull request by benj8560**
## About The Pull Request
Gives service borgs a borg 9mm (like everyone else bruh)
Allows the borg paperwork gripper to grip autolathe disks giving all borgs a means to handle autolathe disks  but not necessarily load materials into them.

Bonus stuff! Reavers now have a 3% chance to drop Reaver Assault armor. I wanted to make it specific the Reavers that wear it but...turns out that code is a little weird and this is the best I can manage. Should avg a 1/3 chance of getting the suit per round assuming all Reavers killed. A rare treat! If you want logic this armor is heavily bomb resistant. Perhaps some faulty implants not being high yield enough to destroy it and just turning the Reaver inside it into jelly. Also a thank you to CDB for helping me figure out how to do this.

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: maps/__Nadezhda/Map_Nadezhda.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #include "../__DeepTunnels/deeptunnels.dm"
 #include "nadezhda.dm"
-#include "../_HunterLodge/lodge.dm"
-#include "../_DeepForest/deepforest.dm"
+//#include "../_HunterLodge/lodge.dm"
```
Comment (Trilbyspaceclone) [inline maps/__Nadezhda/Map_Nadezhda.dm:3]: i dont think you meant to keep these disabled

Comment (benj8560) [reply maps/__Nadezhda/Map_Nadezhda.dm]: remember to give foke a wapping 5 minutes or so to double check are work CHILL feline chill I see it

Comment (Trilbyspaceclone) [reply maps/__Nadezhda/Map_Nadezhda.dm]: if you wanted extra wrap up time then mark it as draft or do so well seeing changes/making pr
Its not even like `RAWR BEN BAD GRR` it was legit "This seems unintented"

**Comment by cdb-is-not-good** [comment]
Can ya add the ability for service borgs to handle the various kitchen pots/pans in this PR as well and test?

**Comment by benj8560** [comment]
> Can ya add the ability for service borgs to handle the various kitchen pots/pans in this PR as well and test?

right so this is infact in the code already but is disabled and a small comment from I assume Hex stating it needs more work. I can...try? But I assume just flipping the switch is not gonna work

**Pull request merged**